### ISSUE TITLE
Colors of theme selection screen should match selected theme 

### ISSUE THREAD

**Issue by peakvalleytech** [alisonthemonster/GratitudeJournal#188]
I've noticed that the colors in screen for selecting a theme is the same regardless of the current theme. As an enhancement for UX, I think the screen should be updated so that its colors are consistent with the current theme.

**Comment by alisonthemonster** [comment]
Yeah I had kept it neutral in the theme selecting screen so that every theme would stand out against the background. If you can find a way to do this and have the themes stand out then please be my guest :)

**Comment by peakvalleytech** [comment]
One solution I have in mind is to add a black (or another color) border to each box. I'll try it out and you can tell me what you think.
